<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObxTab</class>
 <widget class="QWidget" name="ObxTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="eachGB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Each Onebox - scroll or resize dialog to see more; double-click cells to edit</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0" colspan="2">
       <widget class="QTableWidget" name="obxTbl">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>108</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>35</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>34</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>SN</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Meas rate</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>XA</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>XD</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>AI range (V)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Chan map</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Save chans</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="selBox">
        <property name="title">
         <string>Selected row</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="4">
          <widget class="QSpinBox" name="toSB"/>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="defBut">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="copyAllBut">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Copy To All</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="copyToBut">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Copy To</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>obxTbl</tabstop>
  <tabstop>defBut</tabstop>
  <tabstop>copyAllBut</tabstop>
  <tabstop>copyToBut</tabstop>
  <tabstop>toSB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
